### ISSUE TITLE
Front-end Use

### ISSUE THREAD

**Issue by coder20938** [rubengc/cmb2-field-ajax-search#2]
Hello!

Is it possible to use this on the front-end as a search input for users?

Thanks ;)

**Comment by rubengc** [comment]
Yes, but you will need enqueue scripts with the current version, I need to find a better way to enqueue field scripts on demand

**Comment by anmamtl** [comment]
Amazing!
Is there a guide somewhere that can guide me through it? Or some pointers to start?
Thank you!
